source/hardware/controllers/denon_mc7000.rst

```diff
@@ -77,326 +50,253 @@ thread.
+   "13.", "Master Level Meters", "These LEDs show the Master Audio signal."
+   "14.", ":hwlabel:`MASTER LEVEL`", "Turn to adjust the volume level of the master outputs."
+   "15.", ":hwlabel:`BOOTH LEVEL`", "Turn to adjust the volume level of the booth outputs."
+   "20.", ":hwlabel:`PHONES LEVEL`", "Turn to adjust the volume level of the headphones."
+   "21.", ":hwlabel:`PHONES MIX`", "Adjusts the software’s audio output to the headphones, mixing between the cue output and the master mix output."
+   "22.", ":hwlabel:`SPLIT`", "When this switch is lit, the headphone audio will be 'split' such that all channels sent to cue channel are summed to mono and sent to the left headphone channel and the master mix is summed to mono and sent to the right channel. When the switch is in the Off position, the cue channel and master mix will be “blended” together."
+   "   ", ":hwlabel:`X FADER CONTOUR`", "Turn to adjust the slope of the crossfader curve, to the left for a smooth fade (mixing) or to the right for a sharp cut (scratching."
+
+.. note::
+   The Input source selection, Master Level Meters, :hwlabel:`USB1 / USB2`, :hwlabel:`MASTER LEVEL`, :hwlabel:`BOOTH LEVEL`, :hwlabel:`PHONES LEVEL`, :hwlabel:`PHONES MIX` and :hwlabel:`SPLIT` are handled by the controller.
+   The status of Mixxx will not be changed.
+
+
+Microphone Section
+~~~~~~~~~~~~~~~~~~
+
+.. csv-table::
+   :header: "No.", "Control", "Function"
+   :widths: 10 50 250
+
+   "16.", ":hwlabel:`MIC ON/OFF`", "Press to activate/deactivate the microphones."
+   "17.", ":hwlabel:`MIC LEVEL`", "Turn to adjust the volume levels of the corresponding microphone inputs."
+   "18.", ":hwlabel:`MIC 1 HI`/:hwlabel:`MIC 1 LOW` (left deck)", "Turn to boost or cut the high or low frequencies for Microphone 1."
+   "18.", ":hwlabel:`MIC 2 TONE` (right deck)", "Turn to adjust the tone (high-low) for Microphone 2."
+   "18.", ":hwlabel:`ECHO MIC 1&2` (right deck)", "Turn to adjust the amount of the echo effect for both Microphones."
+   "19.", ":hwlabel:`TALKOVER MIC 1&2` (left deck)", "Press to use the “talkover” feature, which automatically reduces the volume level of the master mix when you speak into the microphones."
+   "19.", ":hwlabel:`SHIFT` + :hwlabel:`TALKOVER MIC 1&2` (left deck)", "Press to activate/deactivate the 'Post' feature. When on, the microphones signal will be sent to the Booth Outputs. When off, the microphones signal will not be routed to the Booth Outputs."
+   "19.", ":hwlabel:`ECHO MIC 1&2` (right deck)", "Press this button to activate or deactivate the echo effect on the microphones’ audio signal."
+
+.. note::
+   All functions are handled by the controller in hardware.
+   The status of Mixxx will not be changed.
+
+
+Deck Section
+~~~~~~~~~~~~
+
+.. csv-table::
+   :header: "No.", "Control", "Function"
+   :widths: 10 90 250
```

Comment:
This is wrong, the sum has to be 100%.

Reply:
Not necessarily. The default is 100% but any values are possible to define a ratio between the column width.